ppoprf/src/ppoprf.rs

```diff
@@ -100,6 +308,15 @@ impl ProofDLEQ {
     RistrettoScalar::from_bytes_mod_order_wide(&out)
   }
 
+  fn serialize_element(element: RistrettoPoint) -> String {
+    let byte_array = element.compress().to_bytes();
+    let string = match str::from_utf8(&byte_array) {
```

Comment:
Idiomatically, it would be better to call `as_bytes()` here. The function consumes the `RistretttoPoint` so it doesn't really matter, but it's good practice to delay creation of new objects as long as possible, so the cost is paid only at the step where it's necessary.